### DISCUSSION TITLE
Waiting to heat

### DISCUSSION THREAD

**Discussion by bitzebraeu** [thorrak/fermentrack#711] [Q&A] [answered]
Is there a way how I can influence the Wait to Heat time?

> ![Screenshot_20230319-082910.png](https://user-images.githubusercontent.com/115485565/226264825-393270d6-2eff-453e-a3ef-a73d565632a3.png)

**Comment by thorrak** [comment, marked as answer]
As of last week, yes! :)

You need to install the BrewPi-ESP8266 [v15-beta5](https://github.com/thorrak/brewpi-esp8266/releases/tag/v15-beta5), but once installed you should be able to update the timings from the controller's web interface. You will not be able to adjust these from within Fermentrack unfortunately, but that functionality will likely come at some point in the future.

**Comment by scracas** [comment]
What is the url to the web interface of controller?

**Reply by thorrak** [reply]
Assuming you’re running v15 beta 5, it should just be the IP address of the controller (which should be shown at boot)
